frontend/src/store.js

```diff
@@ -0,0 +1,29 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+
+Vue.use(Vuex)
+
+export default new Vuex.Store({
+  state: {
+    students: []
+  },
+  mutations: {
+    SET_STUDENTS(state, students) {
```

Comment:
```javascript
you can create enum style variable (less typo error)

const MUTATIONS = Object.freeze({
  SET_STUDENTS: 'SET_STUDENTS'
})

/// use in mutations
  mutations: {
   [MUTATIONS.SET_STUDENTS](state, students) { 
    ....
   }
 }  

/// how to use in commit
commit(MUTATIONS.SET_STUDENTS, data)
```

Reply:
Thanks for the trick! Done and committed also.